src/EditorFeatures/CSharpTest/Formatting/Indentation/FormatterTestsBase.cs

```diff
@@ -156,10 +156,8 @@ public static void TestIndentation(int indentationLine, int? expectedIndentation
                                                      It.IsAny<ITextSnapshot>()))
                 .Returns<SnapshotPoint, PointTrackingMode, PositionAffinity, ITextSnapshot>((p, m, a, s) =>
                 {
-                    var factory = workspace.Services.GetService<IHostDependentFormattingRuleFactoryService>()
-                                    as TestFormattingRuleFactoryServiceFactory.Factory;
 
-                    if (factory != null && factory.BaseIndentation != 0 && factory.TextSpan.Contains(p.Position))
+                    if (workspace.Services.GetService<IHostDependentFormattingRuleFactoryService>() is TestFormattingRuleFactoryServiceFactory.Factory factory && factory.BaseIndentation != 0 && factory.TextSpan.Contains(p.Position))
```

Comment:
This one got a bit long, or honestly why it's doing an as at all since I'd imagine this is tests and that better be the case...?